models/public/mask_rcnn_inception_v2_coco/mask_rcnn_inception_v2_coco.md

```diff
@@ -43,36 +44,41 @@ Image, name: `image_tensor`, shape: [1x800x800x3], format: [BxHxWxC],
 
    Expected color order: BGR.
 
-2. Information of input image size, name: `image_info`, shape: [1x3], format: [BxC],
+2. Information about input image size, name: `image_info`, shape: [1x3], format: [BxC],
    where:
 
     - B - batch size
-    - C - vector of 3 values in format [H,W,S], where H is an image height, W is an image width, S is an image scale factor (usually 1)
+    - C - vector of 3 values in format [H,W,S], where H is height, W is width, S is a scale factor (usually 1)
 
 ## Output
 
 ### Original Model
 
-1. Classifier, name: `detection_classes`. Contains predicted bounding boxes classes in a range [1, 91]. The model was trained on the Microsoft\* COCO dataset version with 90 categories of objects, 0 class is for background.
+1. Classifier, name: `detection_classes`. Contains predicted bounding boxes classes in a range [1, 91].
+The model was trained on the Microsoft\* COCO dataset version with 90 categories of objects, 
+0 class is for background.
 2. Probability, name: `detection_scores`. Contains probability of detected bounding boxes.
-3. Detection box, name: `detection_boxes`. Contains detection boxes coordinates in a format `[y_min, x_min, y_max, x_max]`, where (`x_min`, `y_min`)  are coordinates of the top left corner, (`x_max`, `y_max`) are coordinates of the right bottom corner. Coordinates are rescaled to input image size.
+3. Detection box, name: `detection_boxes`. Contains detection boxes coordinates in a format 
+`[y_min, x_min, y_max, x_max]`, where (`x_min`, `y_min`)  are coordinates of the top left corner, 
+(`x_max`, `y_max`) are coordinates of the right bottom corner. Coordinates are rescaled to input image size.
 4. Detections number, name: `num_detections`. Contains the number of predicted detection boxes.
 5. Segmentation mask, name: `detection_masks`. Contains segmentation heatmaps of detected objects for all classes for every output bounding box.
 
 ### Converted Model
 
-1. The array of summary detection information, name: `reshape_do_2d`, shape: [N, 7], where N is the number of detected
-bounding boxes. For each detection, the description has the format:
+- The array of summary detection information, name: `reshape_do_2d`, shape: [N, 7], where N is the number of detected
```

Comment:
I don't think it's a good idea to convert this into a bulleted list. The numbering matters here.